src/main/java/com/cloudcoders/gestaca/logic/student/GetStudent.java

```
@@ -12,6 +13,11 @@ public GetStudent(IStudentDAO iStudentDAO) {
   }
 
   public Student getStudent(String dni) {
-    return iStudentDAO.get(dni);
+    try {
+      return iStudentDAO.get(dni);
+    } catch (PersistenceException e) {
+      e.printStackTrace(); //TODO handle exception properly
```

Need to propagate a unexpected exception